Source/Tests/IRLoader.cpp

```diff
@@ -174,6 +174,8 @@ int main(int argc, char **argv, char **const envp)
   FEXCore::Config::AddLayer(std::make_unique<FEX::ArgLoader::ArgLoader>(argc, argv));
   FEXCore::Config::AddLayer(FEXCore::Config::CreateEnvironmentLayer(envp));
   FEXCore::Config::Load();
+  // Ensure the IRLoader runs in 64-bit mode.
```

Comment:
Maybe expand this, just so it has the reason why we need it to be 64-bit

Reply:
Updated comment that it needs to run in 64-bit mode so the IRLoader has correct SRA for top 8 GPRs and XMM registers.